### ISSUE TITLE
Error using the --supervised option

### ISSUE THREAD

**Issue by malbianco** [AI-sandbox/neural-admixture#38]
Hi,

I'm currently using the Version: 1.6.3, and I'm getting this error

```
neural-admixture train --k 10 --supervised --populations_path pop_.txt --name RUN_Supervise --data_path Data/Admixture_Genotype.bed --save_dir ADMX_Training  --num_cpus 15 --num_gpus 1



    _   _                      _       ___  ____  __  __ _______   _________ _    _ _____  ______
   | \ | |                    | |     / _ \|  _ \|  \/  |_   _\ \ / /__   __| |  | |  __ \|  ____|
   |  \| | ___ _   _ _ __ __ _| |    / /_\ | | | | \  / | | |  \ V /   | |  | |  | | |__) | |__
   | . ` |/ _ \ | | | '__/ _` | |    |  _  | | | | |\/| | | |   > <    | |  | |  | |  _  /|  __|
   | |\  |  __/ |_| | | | (_| | |    | | | | |_| | |  | |_| |_ / . \   | |  | |__| | | \ \| |____
   |_| \_|\___|\__,_|_|  \__,_|_|    \_| |_/____/|_|  |_|_____/_/ \_\  |_|   \____/|_|  \_\______|


                                     Version: 1.6.3
                    Authors: Joan Saurina Ricós, Albert Dominguez Mantes,
                            Daniel Mas Montserrat, Alexander G. Ioannidis.
                    Help: https://github.com/AI-sandbox/neural-admixture

    There are 15 CPU's available for this execution. Hence, using 7 threads.
    Operating system is Linux!
    Input format is BED.
    Data contains XXXXX samples and XXXXX SNPs.

    Running SVD...

    1) Generating Ω y Y = A @ Ω...
       Time=19.1133s
       Power iterations time=581.4033s
    2) QR of Y...
       Time=0.2550s
    3) B = Qᵀ @ A...
       Time=261.4448s
    4) SVD of B...
       SVD time=0.0548s

    Total time SVD: 862.2726s

    Entering single-GPU or CPU training...
usage: neural-admixture train [-h] [--epochs EPOCHS] [--batch_size BATCH_SIZE] [--learning_rate LEARNING_RATE] [--seed SEED] [--k K] [--min_k MIN_K] [--max_k MAX_K] [--hidden_size HIDDEN_SIZE] --save_dir SAVE_DIR --data_path DATA_PATH
                              --name NAME [--supervised_loss_weight SUPERVISED_LOSS_WEIGHT] [--pops_path POPS_PATH] [--n_components N_COMPONENTS] [--num_gpus NUM_GPUS] [--num_cpus NUM_CPUS]
neural-admixture train: error: argument --supervised_loss_weight: expected one argument
```

I tried to use the option --supervised_loss_weight 99 but I still get the same error. 
If I run the program using the unsupervised version, I don't have any problem.

**Comment by joansaurina** [comment]
Hi @malbianco,

You are absolutely right — we removed the `--supervised` flag in the latest release, but it was mistakenly still documented on the GitHub page. Thank you for pointing that out.

To run in supervised mode, it is sufficient to provide the `--pops_path `argument. If you would like to adjust the contribution of the supervised loss, you can also include the `--supervised_loss_weight` parameter. For example:

```
neural-admixture train \
  --k 10 \
  --pops_path pop_.txt \
  --supervised_loss_weight 100 \
  --name RUN_Supervise \
  --data_path Data/Admixture_Genotype.bed \
  --save_dir ADMX_Training \
  --num_cpus 15 \
  --num_gpus 1
```
We'll update the documentation as soon as possible to reflect this change.

Best regards,
Joan
